src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/OrganisationalInternalControllerProviderTest.java

```diff
@@ -129,7 +141,7 @@ public void setUpUserForInviteToOrganisation() throws IOException {
         when(professionalUserService
             .findProfessionalUserByEmailAddress(anyString())).thenReturn(pu);
 
-        when(prdEnumService.getPrdEnumByEnumType(any())).thenReturn(Arrays.asList("role"));
+        when(prdEnumService.getPrdEnumByEnumType(any())).thenReturn(asList("role"));
```

Comment:
Please use `List.of()` to return a list instead of `Arrays.asList()`

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/b17549eb7a6a39bc04a0482a299739fe4dc1e212